bundledApi.yaml

```diff
@@ -37,6 +37,34 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/error'
+  /models/{identifier}:
```

Comment:
You suppose to add all this in openapi and not in bundledApi. The bundledApi file is generated according to openapi.
Next change we will do in openapi results in generating new file that will override the bundleApi..
Move all the changes to openapi and run the bundleApi command so the bundleApi will change according to it

Reply:
fixed